tree/ntuple/v7/inc/ROOT/RNTupleOptions.hxx

```diff
@@ -96,6 +98,7 @@ public:
 // clang-format on
 class RNTupleWriteOptionsDaos : public RNTupleWriteOptions {
   std::string fObjectClass{"SX"};
+  uint32_t fCageSizeLimit = 16 * RNTupleWriteOptions::fApproxUnzippedPageSize;
```

Comment:
Perhaps we should add a comment arguing that we experimentally measured that 1MB yields acceptable results in most use cases.